flocker/apiclient/_client.py

```diff
@@ -536,3 +556,17 @@ def to_nodes(result):
         request.addCallback(to_nodes)
 
         return request
+
+    def this_node_uuid(self):
+        era = check_output(["flocker-node-era"])
```

Comment:
Possibly this should use a Twisted API.

On the other hand it only needs to be called once per process, at startup time.